plugins/pipelines/README.md

```diff
@@ -107,9 +107,9 @@ Only PR events of specified PR number will be searched when `prNum` is set
 `GET /pipelines/{id}/events?page={pageNumber}&count={countNumber}&sort={sort}&prNum={prNumber}`
 
 #### Get all jobs (including pull requests jobs)
-`archived` is optional and has a default value of `false`, which makes the endpoint not return archived jobs (e.g. closed pull requests)
+`archived` is optional and has a default value of `false`, which makes the endpoint not return archived jobs (e.g. closed pull requests). `type` is optional and can be set to `pr` to only return PR jobs.
 
-`GET /pipelines/{id}/jobs?archived={boolean}`
+`GET /pipelines/{id}/jobs?archived={boolean}&type=pr`
```

Comment:
What about having a param as per our original discussion?
`v4/pipelines/:id/jobs?pr=true`
`pr=true` returns all jobs in the pipeline that belong to a PR
`pr=false` returns all jobs in the pipeline that belong to the main branch of the SCM repo
when the `pr` parameter is not set, the response should be to return all jobs for the pipeline